tests/h/presenters_test.py

```diff
@@ -32,6 +32,28 @@ def test_uri_returns_Markup(self):
         assert isinstance(self._annotation(target_uri="http://foo.com").uri,
                           jinja2.Markup)
 
+    def test_quote(self):
+        annotation = self._annotation(
+            annotation=mock.Mock(
+                target_selectors=[{'exact': 'selected text'}],
+                text="entered text"
+            )
+        )
+
+        assert annotation.quote == (
```

Comment:
You don't need to split an assert over multiple lines if it is <= 79 chars total